analysis/metric_space.lean

```diff
@@ -386,5 +386,30 @@ have tendsto (λp:α×α, dist p.1 p.2) (nhds (a, b)) (nhds (dist a b)),
   from continuous_iff_tendsto.mp continuous_dist' (a, b),
 (hf.prod_mk hg).comp (by rw [nhds_prod_eq] at this; exact this)
 
+lemma tendsto_iff_dist_tendsto_zero { X Y : Type*} [metric_space X] [metric_space Y]
+{f : X → Y} {x : X} {y : Y}: (tendsto f (nhds x) (nhds y)) ↔ (tendsto (λ x', dist (f x') y) (nhds x) (nhds 0)) :=
+begin
+split,
+{ intro lim,
+  have := tendsto_dist lim tendsto_const_nhds, swap, exact y,
+  finish[this] },
```

Comment:
Please don't write multiple tactics in one line!
Especially when it is something like `swap`.